.github/workflows/examples-nrfconnect.yaml

```diff
@@ -40,6 +40,8 @@ jobs:
               with:
                   fetch-depth: 0
                   submodules: true
+            - name: Setup environment by running activate script.
+              run: source scripts/activate.sh
```

Comment:
Looks like you need to combine it with the next step using "&&".

Reply:
Currently the problem is that scripts are ran in shell not bash, so source command is not found, but I'm working on it.